models/google_ads.yml

```diff
@@ -56,4 +56,6 @@ models:
       - name: criteria_id
         description: "{{ doc('criteria_id') }}"
       - name: gclid
-        description: "{{ doc('gcl_id') }}"
\ No newline at end of file
+        description: "{{ doc('gcl_id') }}"
+      - name: rn
+        description: Row number partitioned by gclid and ordered by date_day which is used to dedupe possible double gclids.
```

Comment:
Not sure this field is needed in the final model. However, I thought it would be safer to keep it and add a description opposed to removing it.